app/templates/brief.html

```diff
@@ -52,15 +52,11 @@
 {% elif brief.status == 'cancelled' %}
 <div class="grid-row">
   <div class="column-one-whole">
-    {% set brief_cancelled_message %}
-      The buyer cancelled this opportunity, for example because they no longer have the budget.<br />
-      They may publish an updated version later.
-    {% endset %}
     {%
       with
       type = "temporary-message",
       heading = "This opportunity was cancelled",
-      message = brief_cancelled_message
+      message = The buyer cancelled this opportunity, for example because they no longer have the budget. They may publish an updated version later.
```

Comment:
Does it not need the `<br />` now?
Also If the heading is going to have quote then it would be nice if the message did too. Looks like the message does in other places.

Otherwise looks good

Reply:
Thanks, yes, I forgot the quotes at first, now fixed :).

Removing line separators is the main thing about this PR, as all message should be on one line apparently, so no <br /> ☺️

Reply:
👍